editor.planx.uk/src/@planx/components/List/Public/Fields.tsx

```diff
@@ -145,12 +146,19 @@ export const RadioFieldInput: React.FC<Props<QuestionField>> = ({
   );
 };
 
-export const SelectFieldInput: React.FC<Props<QuestionField>> = ({
-  id,
-  data,
-  required,
-}) => {
+export const SelectFieldInput: React.FC<Props<QuestionField>> = (props) => {
   const { formik, activeIndex } = useListContext();
+  const { id, data, required } = props;
+
+  const isDisabled = (option: Option) => {
```

Comment:
This is just visual feedback, it would be nice to also incorporate this into the schema, most likely with a yup test ([docs](https://github.com/jquense/yup?tab=readme-ov-file#schematestoptions-object-schema)).